src/Api/ResetPasswordApiController.php

```diff
@@ -1,20 +1,20 @@
 <?php
 
-namespace Saritasa\Laravel\Controllers\Api;
+namespace Saritasa\LaravelControllers\Api;
 
 use Dingo\Api\Http\Response;
 use Illuminate\Contracts\Auth\Authenticatable;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Foundation\Auth\ResetsPasswords;
 use Dingo\Api\Http\Request;
 use Illuminate\Support\Str;
-use Saritasa\Laravel\Controllers\Responses\ErrorMessage;
-use Saritasa\Laravel\Controllers\Responses\SuccessMessage;
+use Saritasa\LaravelControllers\Responses\ErrorMessage;
+use Saritasa\LaravelControllers\Responses\SuccessMessage;
 
 /**
  * This controller is responsible for handling password reset requests
  * Utilize native Laravel password management without UI, in API style
- * https://laravel.com/docs/5.4/passwords
+ * https://laravel.com/docs/5.6/passwords
```

Comment:
Better is without laravel version: https://laravel.com/docs/passwords
Then it will be redirected to newest version